CONTRIBUTING.md

```diff
@@ -155,7 +155,6 @@ We use [CSS Modules](https://github.com/css-modules/css-modules) along with [BEM
 To maintain React 16.4 compatibility, please do not use newer features of React such as Hooks.
 
 When creating (S)CSS files, follow the CSS Module naming convention by using the `.module.(s)css` extension.
-
```

Comment:
Think this should still be here to ensure correct processing of headers?

Reply:
Yeah I'm not sure how that got removed! I'll add it back in